base/applications/utilman/umandlg/about.c

```diff
@@ -43,12 +43,7 @@ INT_PTR CALLBACK AboutDlgProc(HWND hDlg, UINT Msg, WPARAM wParam, LPARAM lParam)
     {
         case WM_INITDIALOG:
         {
-            Globals.hIcon = LoadImageW(Globals.hInstance,
-                                       MAKEINTRESOURCEW(IDI_ICON_UTILMAN),
-                                       IMAGE_ICON,
-                                       0,
-                                       0,
-                                       LR_DEFAULTSIZE);
+            Globals.hIcon = ExtractIconW(Globals.hInstance, L"utilman.exe", 0);
```

Comment:
Should this be the exename of the application loading this dll?
Or should it only be `utilman.exe` found somewhere on the path?

Reply:
> Should this be the exename of the application loading this dll?

It's the application process that loads this DLL, yes.

Reply:
In that case, something like this would be more appropriate:
```
WCHAR fullPath[MAX_PATH];
GetModuleFileNameW(NULL, fullPath, ARRAY_SIZE(fullPath);
Globals.hIcon = ExtractIconW(Globals.hInstance, fullPath, 0);
```